### PR TITLE
upgrade preinstalled packages in final build step

### PR DESCRIPTION
**Pull request by Zoey2936**
close #4491

### CODE REVIEW

File: Containers/imaginary/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:latest
 FROM golang:1.22.2-alpine3.18 as go
 
-ENV IMAGINARY_HASH 6cd9edd1d3fb151eb773c14552886e4fc8e50138
+ENV IMAGINARY_HASH 6cd9edd1d3fb151eb773c14552886e4fc8e50138
 
 RUN set -ex; \
     apk add --no-cache \
```
Comment (szaimen) [inline Containers/imaginary/Dockerfile:7]: here as well or rather not?

File: Containers/talk/Dockerfile [hunk 2]
```diff
@@ -46,6 +46,7 @@ COPY --chmod=775 healthcheck.sh /healthcheck.sh
 COPY --chmod=664 supervisord.conf /supervisord.conf
```
Comment (szaimen) [inline Containers/talk/Dockerfile:46]: and in the janus build as well?

File: Containers/imaginary/Dockerfile (hunk 1 above)
Comment (Zoey2936) [reply Containers/imaginary/Dockerfile:7]: The preinstalled packages aren't statucally linked during build, so I think it isn't needed

File: Containers/talk/Dockerfile (hunk 2 above)
Comment (Zoey2936) [reply Containers/talk/Dockerfile:46]: The preinstalled packages aren't statucally linked during build, so I think it isn't needed

File: Containers/imaginary/Dockerfile (hunk 1 above)
Comment (szaimen) [reply Containers/imaginary/Dockerfile:7]: ok

File: Containers/talk/Dockerfile (hunk 2 above)
Comment (szaimen) [reply Containers/talk/Dockerfile:46]: ok

**Review by szaimen — APPROVED** [review]
LGTM, thanks! :)

**Comment by szaimen** [comment]
This is now released with v8.2.1 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**